### ISSUE TITLE
FR: Timestamp with headphones (on mobile)

### ISSUE THREAD

**Issue by SeLiEnisyst** [chhoumann/PodNotes#95]
Hi there,

Firstly, I want to express my gratitude for developing such an amazing plugin! It has significantly enhanced my note-taking experience while listening to podcasts on my desktop.

However, I often find myself listening to podcasts outdoors, wearing headphones. In these situations, when I come across something noteworthy, it would be incredibly convenient if I could simply tap on one of my earbuds to create a timestamp, for being able to directly jump to that point when I'm back at my desktop.

I've noticed that the Android app Snipd offers a similar feature, allowing users to configure the app to create timestamps instead of jumping to the previous title.

Integrating this functionality into your plugin would undoubtedly elevate its usability for mobile users like myself, enabling seamless note-taking even while on the go.

Thank you for considering my suggestion!

Best regards,
Silias

**Comment by p33k** [comment]
maybe the "easiest" way would be the option to add a timestamp (on mobile) when the player is paused and started again?
that way it would work on Android, iOs and so on with every earbud when you just stop and start the podcast with the earbud.

**Comment by SeLiEnisyst** [comment]
Yeah, that could work. But the downside of this would be, that it would ceate too mutch unwanted timestamps…

 

Bei Rückfragen stehe ich jederzeit gerne zur Verfügung.

 

Mit freundlichen Grüßen / Best Regards

Sebastian Liebel

 

Mobil: +49-176-15317537

 

Von: p33k ***@***.***> 
Gesendet: Montag, 20. Mai 2024 11:34
An: chhoumann/PodNotes ***@***.***>
Cc: Sebastian Liebel ***@***.***>; Author ***@***.***>
Betreff: Re: [chhoumann/PodNotes] FR: Timestamp with headphones (on mobile) (Issue #95)

 

[External Mail] 

maybe the "easiest" way would be the option to add a timestamp (on mobile) when the player is paused and started again?
that way it would work on Android, iOs and so on with every earbud when you just stop and start the podcast with the earbud.
